carmajava/cooperativemerge/src/main/java/gov/dot/fhwa/saxton/carma/plugins/cooperativemerge/ExecutionState.java

```diff
@@ -39,7 +39,14 @@
  * Speed and steering commands received from an rsu are sent to the complex maneuver for execution.
  */
 public class ExecutionState implements ICooperativeMergeState {
-  
+
+  protected static final double MIN_TARGET_SPEED = 0.5; // m/s - if traffic is heavy, a slow speed may be okay
+  protected static final double MAX_TARGET_SPEED = 35.0; // m/s - a little over 75 mph
+  protected static final double MIN_ACCEL = -2.0; // m/s^2 - trying to keep it smooth regardless of vehicle type
+  protected static final double MAX_ACCEL = 2.0;  // m/s^2
+  protected static final double MIN_TARGET_STEER = -1.0; //currently not used; what units? what meaning?
+  protected static final double MAX_TARGET_STEER = 1.0; //currently not used; what units? what meaning?
```

Comment:
Steer in this case refers to rad and is how we trigger the lane change indicator on the UI in CARMA2

Reply:
Updated comments.